Tasks/src/V2/HttpRequest.php

```diff
@@ -371,6 +376,11 @@ public function getOidcToken()
         return $this->readOneof(6);
     }
 
+    public function hasOidcToken()
+    {
+        return $this->hasOneof(6);
```

Comment:
This, and many of the other new `hasXXX` methods introduced in this PR all throw fatal errors when called e.g. `Error: Call to undefined method  Google\Cloud\Tasks\V2\HttpRequest::hasOneof()`.

This is the case with both the native extension and composer protobuf implementations of protobuf.

Looks like the underlying `hasOneOf` method was introduced in protobuf 3.13.0 but [never actually made it into the release](https://github.com/protocolbuffers/protobuf-php/issues/7) for packagist or pecl.

Therefore all these methods are 100% broken until a newer version of protobuf is released.

Obviously seems there's an issue with the protobuf release process, but perhaps the google-cloud-php build/test tooling also needs to be updated to run some sanity checks so that it's not possible to release a wrapper with broken methods in future?

Reply:
Thanks for the heads up. We'll work on getting that addressed.